docs/pom.xml

```diff
@@ -23,7 +23,7 @@
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
         <version>1.0.7</version>
```

Comment:
Does this match the version in file? I'd prefer to see this updated rather than have `sed` in Jenkins job.

Reply:
- https://github.com/eclipse-ee4j/glassfish/pull/24181

Reply:
There;s already a bash script for that, updateVersions ... however I did not run it for a while, so maybe it would need some update.